pkg/plugins/handlers.go

```
@@ -168,11 +168,18 @@ func (p *PluginsHandler) proxyPluginRequest(requestURL *url.URL, pluginName stri
 	if err != nil {
 		errMsg := fmt.Sprintf("failed to send GET request for %q plugin: %v", pluginName, err)
 		klog.Error(errMsg)
-		serverutils.SendResponse(w, http.StatusBadGateway, serverutils.ApiError{Err: errMsg})
+		serverutils.SendResponse(w, http.StatusServiceUnavailable, serverutils.ApiError{Err: errMsg})
 		return
 	}
 	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)